cmd/cloud/remote/client.go

```diff
@@ -68,10 +69,25 @@ $ ory ...
 	}
 }
 
+func GetProjectSlug(cmd *cobra.Command) (string, error) {
+	url := flagx.MustGetString(cmd, FlagEndpoint)
+	client := NewHTTPClient(cmd)
+
+	rsp, err := client.Get(fmt.Sprintf("%s/projects/%s/slug", url, os.Getenv(projectEnvKey)))
```

Comment:
`projectEnvKey` is `ORY_PROJECT_ID` which this PR is aiming to make obsolete. This should be a HTTP request like so:

```
HTTP GET 1.1 https://api.backofice.../tokens/slugs

Authorization: Bearer <ory-access-token>
```

The result is

```
{ "slug": "asdf" }
```